mongo/collection.go

```diff
@@ -311,7 +322,15 @@ func (coll *Collection) InsertOne(ctx context.Context, document interface{},
 	return &InsertOneResult{InsertedID: res[0]}, err
 }
 
-// InsertMany inserts the provided documents.
+// InsertMany performs an insert operation (https://docs.mongodb.com/manual/reference/command/insert/) to insert
+// multiple documents into the collection. If write errors occur during the operation (e.g. duplicate key error),
+// this method returns a BulkWriteException error.
+//
+// The documents parameter must be a slice of documents to insert. The slice cannot be nil or empty. The elements must
+// all be non-nil. For any document that does not have an _id field when transformed into BSON, one will be added
+// automatically to the marshalled document. The original document will not be modified.
```

Comment:
Mention that you can retrieve the _id values from the InsertManyResult

Reply:
done.